pbspro.spec.in

```diff
@@ -66,7 +66,7 @@
 %define pbs_dist %{pbs_name}-%{pbs_version}.tar.gz
 
 %if !%{defined _unitdir}
-%define _unitdir /usr/lib/systemd/system
+%define _unitdir @_unitdir@
```

Comment:
Please apply this change to pbspro.spec as well. We need to keep them in sync.

Reply:
Thanks @mike0042 , changes added to the pbspro.spec file as well. Please review it.